querybook/webapp/ui/Modal/FullScreenModal.tsx

```diff
@@ -16,22 +17,17 @@ export const FullScreenModal: React.FunctionComponent<IStandardModalProps> = ({
         [className]: Boolean(className),
     });
 
-    let modalTitleDOM: React.ReactNode;
-    if (!hideModalTitle) {
-        modalTitleDOM = (
-            <>
-                <IconButton
-                    className="Modal-close"
-                    aria-label="close"
-                    icon="x"
-                    onClick={onHide}
-                />
-                {title !== null ? (
-                    <div className="Modal-title">{title}</div>
-                ) : null}
-            </>
-        );
-    }
+    const modalTitleDOM = (
+        <>
+            <IconButton
+                className="Modal-close"
+                aria-label="close"
+                icon="x"
+                onClick={onHide}
+            />
+            {title !== null ? <div className="Modal-title">{title}</div> : null}
```

Comment:
i just realized its probably better if we have title != null just in case its undefined